### PR TITLE
UID2-3726 Add instructions on scraping metrics for Azure

### PR DESCRIPTION
**Pull request by cYKatherine**

### CODE REVIEW

File: docs/guides/operator-guide-azure-enclave.md [hunk 1]
```diff
@@ -114,6 +114,12 @@ There are some limitations with regard to location:
 
 - For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-region-availability#linux-container-groups) in the Azure documentation. You can only deploy to regions with the **Confidential SKU** column set to **Y** in the table.
```
Comment (cYKatherine) [inline docs/guides/operator-guide-azure-enclave.md:115]: @thomasm-ttd FYI I checked this page and I think it's updated, there is no "Confidential SKU" column/tables anymore. Should we consider removing this section? Do we still need to mention `Confidential SKU`?

Comment (thomasm-ttd) [reply docs/guides/operator-guide-azure-enclave.md:115]: The link has changed to : https://learn.microsoft.com/en-us/azure/container-instances/container-instances-resource-and-quota-limits#confidential-container-resources-preview
Change the text to: 
For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-resource-and-quota-limits#confidential-container-resources-preview) in the Azure documentation to confirm the availability of Confidential Container regional support.

File: docs/guides/operator-guide-azure-enclave.md [hunk 2]
```diff
@@ -286,6 +292,9 @@ import AttestFailure from '/docs/snippets/_private-operator-attest-failure.mdx';
 
 <AttestFailure />
 
+### Scraping Metrics
+The Private Operator for Azure exposes [Prometheus-formatted metrics](https://prometheus.io/docs/concepts/data_model/) on port 9080 through the /metrics endpoint. You can use a Prometheus-compatible scraper to collect and aggregate these metrics for your own needs.
```
Comment (thomasm-ttd) [inline docs/guides/operator-guide-azure-enclave.md:296]: Suggest we say:
You can use a Prometheus-compatible scraper to collect and aggregate these metrics for your own needs. This scraper will need to have access to the VNet that the operators are running in. It is not recommended to allow access the /metrics endpoint from the load balancer.

File: docs/guides/operator-guide-azure-enclave.md [hunk 3]
```diff
@@ -286,6 +292,11 @@ import AttestFailure from '/docs/snippets/_private-operator-attest-failure.mdx';
 
 <AttestFailure />
 
+### Scraping Metrics
+The Private Operator for Azure exposes [Prometheus-formatted metrics](https://prometheus.io/docs/concepts/data_model/) on port 9080 through the `/metrics` endpoint. You can use a Prometheus-compatible scraper to collect and aggregate these metrics for your own needs. 
+
+This scraper will need to have access to the VNet that the operators are running in. It is not recommended to allow accessing the `/metrics` endpoint from the load balancer.
```
Comment (genwhittTTD) [inline docs/guides/operator-guide-azure-enclave.md:298]:
```suggestion
The scraper must have access to the VNet that the Private Operator is running in. We do not recommend giving the load balancer access to the `/metrics` endpoint.
```

**Review by genwhittTTD — CHANGES_REQUESTED** [review]
Left 2 comments. Thanks!

**Review by genwhittTTD — APPROVED** [review]
LGTM thx.

**Pull request merged**